src/Bundle/Resources/config/services/form.xml

```diff
@@ -6,9 +6,10 @@
     <services>
         <defaults public="true" />
 
-        <service id="sylius.form.type.resource_autocomplete_choice" class="Sylius\Bundle\ResourceBundle\Form\Type\ResourceAutocompleteChoiceType">
+        <service id="Sylius\Bundle\ResourceBundle\Form\Type\ResourceAutocompleteChoiceType">
             <argument type="service" id="sylius.registry.resource_repository" />
             <tag name="form.type" />
         </service>
+        <service id="sylius.form.type.resource_autocomplete_choice" alias="Sylius\Bundle\ResourceBundle\Form\Type\ResourceAutocompleteChoiceType" />
```

Comment:
Is this change necessary? Autowire should already work with previous definition, shouldn't it?

Reply:
Indeed, this change is not for autowiring. new services in Sylius uses FQCN for service id or the interface. So it's to slightly moves to FQCN ids.